README.md

```diff
@@ -97,8 +97,10 @@ Regulated non-standard definitions\footnote{%
 A GNU/Linux-based operating system is assumed.
 The described instructions may be valid for other operating systems but this is not guaranteed.
 
-In order to compile the document, install TexLive (Debian-based APT packages: `texlive-full lyx`)
-and the Python packages listed in `requirements.txt`.
+In order to compile the document, install TexLive (Debian-based APT packages: `texlive-full lyx`),
```

Comment:
This isn't helpful. I spent about 15 mintues trying to figure out how to map `texlive-full` to the verison of pdfLaTex but ubuntu doesn't list pdflatex versions in their manifests. You seem to be able to build this document. What are you using?

Reply:
It should be better now. I am using this: https://packages.ubuntu.com/xenial/texlive-full. Upon a closer look, fixing the container at Ubuntu 16.04 should work. There are also prebuilt containers out there shipped with specific versions of LaTeX googleable as `texlive container`.